team_AlphaPwners/index.md

```diff
@@ -0,0 +1,37 @@
+# AlphaPwners Cookies
+### by team AlphaPwners
+
+```Python
+# Import elements
+import ingrediends from fridge as ing
+import microwave, bowl, utils from kitchen
+
+# Add ingredients into bowl
+bowl.add(ing.SUGAR, 4)
+bowl.add(ing.FLAVOUR, 4)
+bowl.add(ing.CACAO, 2)
+bowl.add(ing.OIL, 1)
+bowl.add(ing.MILK, 3)
+bowl.add(ing.EGG, 1)
+bowl.add(ing.BAKING_POWDER, 0.25)
+bowl.add(ing.SALT, 0.05)
+
+# Mixing and baking
+bowl.mix_together()
+microwave.bake(bowl, 150)
```

Comment:
150 watts? Minutes?

Reply:
You should check the kitchen.microwave docs :D Its fine or i really need to fix this. Some explainations are in film.